src/praisonai/praisonai/cli/main.py

```
@@ -581,7 +560,8 @@ def __init__(self):
         # They no longer open Chainlit browser UI
 
         if getattr(args, 'realtime', False):
-            self.create_realtime_interface()
+            from praisonai.cli.commands.ui import _launch_aiui_app
+            _launch_aiui_app("ui_realtime", "ui_realtime", 8085, "0.0.0.0", None, False, "Realtime Voice")
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)